src/Cms/Language.php

```diff
@@ -210,6 +210,9 @@ public static function create(array $props)
 			$props['default'] = true;
 		}
 
+		// trigger before hook
+		$kirby->trigger('language.create:before', compact('props'));
+
 		$language = new static($props);
```

Comment:
I think we should use the same parameters as we do in Page::create. 

```suggestion
		$language = new static($props);

		$kirby->trigger('language.create:before', ['language' => $language, 'input' => $props]);
```